test/configs/zombieDataPreservers.json

```diff
@@ -0,0 +1,114 @@
+                "validator": true
+            }
+        ]
+    },
+    "parachains": [
+        {
+            "id": 1000,
+            "chain_spec_path": "specs/single-container-tanssi-1000.json",
+            "COMMENT": "Important: these collators will not be injected to pallet-invulnerables because zombienet does not support that. When changing the collators list, make sure to update `scripts/build-spec-single-container.sh`",
+            "collators": [
+                {
+                    "name": "FullNode-1000",
+                    "validator": false,
+                    "command": "../target/release/tanssi-node",
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"],
+                    "ws_port": 9948
+                },
+                {
+                    "name": "Collator1000-01",
+                    "command": "../target/release/tanssi-node",
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"]
+                },
+                {
+                    "name": "Collator1000-02",
+                    "command": "../target/release/tanssi-node",
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"]
+                },
+                {
+                    "name": "Collator2000-01",
+                    "command": "../target/release/tanssi-node",
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"]
+                },
+                {
+                    "name": "Collator2000-02",
+                    "command": "../target/release/tanssi-node",
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"]
+                },
+                {
+                    "name": "Collator1000-03",
+                    "command": "../target/release/tanssi-node",
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"]
+                }
+            ]
+        },
+        {
+            "id": 2000,
+            "chain_spec_path": "specs/single-container-template-container-2000.json",
+            "collators": [
+                {
+                    "name": "FullNode-2000",
+                    "validator": false,
+                    "command": "../target/release/container-chain-simple-node",
+                    "args": ["--no-hardware-benchmarks", "--database=paritydb", "--wasmtime-precompiled=wasm"],
+                    "ws_port": 9949,
+                    "p2p_port": 33049,
+                    "prometheus_port": 33102
+                },
+                {
+                    "name": "DataPreserver",
+                    "validator": false,
+                    "command": "scripts/container-chain-args.sh --rpc-port 9950 -- ../target/release/container-chain-simple-node",
```

Comment:
Not sure if you need this script in the end. If you only want to add some container args, you can do it like `keep-db` in this test:

https://github.com/moondance-labs/tanssi/blob/57a05db22eb408aa3d6bab146f38c1fbb128df23/test/configs/zombieTanssiKeepDb.json#L77C26-L77C37

So I guess `"-- --rpc-port 9950"`

The reason why container args come before relay chain args is because that's the order you get when you add `-- --arg` here in zombienet. Originally I tried to add container args after relay args but that didn't work.